make.py

```
@@ -51,7 +52,20 @@ def enablePrint():
     enablePrint()
 
 import sck
-kit = sck.sck(check_pio_sam='sam' in sys.argv, check_pio_esp='esp' in sys.argv)
+
+if 'dump-env' in sys.argv:
+    other_actions = [a for a in ('build', 'flash', 'boot') if a in sys.argv]
+    if other_actions:
+        ERROR('dump-env cannot be combined with other actions: ' + ', '.join(other_actions)); sys.exit()
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)